lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -200,6 +201,99 @@ class TripBasicsPane extends Component<TripBasicsProps, State> {
     }
   }
 
+  RenderAvailableDays = ({
```

Comment:
Move this component to top-level (it doesn't look like you are using `this`)

Reply:
Fixed along with the rest of feedback in [5b0b986](https://github.com/opentripplanner/otp-react-redux/pull/1280/commits/5b0b98662afc215373192ea41086301bef40e224)!! Thanks